ios/FluentUI/Label/Label.swift

```diff
@@ -64,24 +39,6 @@ open class Label: UILabel, TokenizedControlInternal {
             updateTextColor()
         }
     }
-    private var _textColor: UIColor?
-
-    public typealias TokenSetKeyType = EmptyTokenSet.Tokens
-    public var tokenSet: EmptyTokenSet = .init()
-
-    private var isUsingCustomAttributedText: Bool = false
-
-    @objc public init(style: AliasTokens.TypographyTokens = .body1, colorStyle: TextColorStyle = .regular) {
-        self.style = style
-        self.colorStyle = colorStyle
-        super.init(frame: .zero)
-        initialize()
-    }
-
-    @objc public required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
-        initialize()
-    }
 
     open override func willMove(toWindow newWindow: UIWindow?) {
```

Comment:
is just updating the color fomr the willmovewindow sufficient or do we need to updatefonts in general?